src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
```

Comment:
The function should handle cases where `sourceString` is empty or null to avoid runtime errors. Consider adding a check at the beginning of the function to return an empty object if `sourceString` is falsy.